.github/workflows/codeql.yaml

```diff
@@ -66,15 +67,28 @@ jobs:
       id: codeql_analysis
       uses: github/codeql-action/analyze@v2
 
-    - name: Send message to slack channel
-      id: notify
-      if: ${{ always() &&  github.event_name == 'schedule'}}
+    - name: Send message to helm_dev slack channel
+      id: notify_dev
+      if: ${{ always() &&  github.event_name == 'schedule' && steps.codeql_analysis.conclusion != 'success' }}
```

Comment:
You have the checks reversed. this should be == "success" and failure message should be != sucsess